frontend/src/components/navbar/Sidebar.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  Box,
+  Drawer,
+  IconButton,
+  List,
+  ListItem,
+  ListItemText,
+} from "@mui/material";
+import { useState } from "react";
+import { navbarItem } from "./nav";
+import { NavLink } from "react-router-dom";
+import MenuIcon from "@mui/icons-material/Menu";
+
+const Sidebar = () => {
+  const [drawerOpen, setDrawerOpen] = useState(false);
+
+  const toggleDrawer = (open: boolean) => () => {
+    setDrawerOpen(open);
+  };
+
+  const drawer = (
```

Comment:
You still haven't changed the drawer naming. This is not the recommended name for jsx components.

Reply:
Example

Reply:
you should be familiar with it i guess. anyways here is the link for you to read:

https://www.linkedin.com/pulse/react-js-naming-convention-kristiyan-velkov/